From: Jon Severinsson
Date: Mon, 2 Apr 2014 15:00:00 +0200
Subject: Map SysV runlevel S to basic.target, not sysinit.target.

Debian specific change, upstream does not map runlevel S to any target.
--- a/src/core/service.c
+++ b/src/core/service.c
@@ -68,7 +68,7 @@
         { "rc5.d",  SPECIAL_RUNLEVEL5_TARGET, RUNLEVEL_UP },
 
         /* Debian style rcS.d */
-        { "rcS.d",  SPECIAL_SYSINIT_TARGET,   RUNLEVEL_SYSINIT },
+        { "rcS.d",  SPECIAL_BASIC_TARGET,   RUNLEVEL_SYSINIT },
 
         /* Standard SysV runlevels for shutdown */
         { "rc0.d",  SPECIAL_POWEROFF_TARGET,  RUNLEVEL_DOWN },
